exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/data/data_flights.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
<p>Can we use jupysql here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/306/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
We can. I just used pyexasol to make it consistent with other notebooks in the data importing section.

Reply:
Changed to jupysql.